gulpfile.js

```diff
@@ -227,9 +266,15 @@ gulp.task('instrumentForCoverage', ['build'], function(done) {
         console.log(stdout);
         done();
     });
-});
+}));
+
+gulp.task('release', gulp.series('generateStubs', function() {
+    return combine()
```

Comment:
Is there a reason we can't just pass all the functions to `gulp.series`?
ie `gulp.series('generateStubs', combine, minifyRelease, generateDocumentation)`

Reply:
Good idea.  I just didn't notice it when changing the tasks. Done